ckanext/datasetapproval/plugin.py

```diff
@@ -57,16 +58,33 @@ def get_auth_functions(self):
         }
 
     def get_dataset_labels(self, dataset_obj):
-        labels = super(DatasetapprovalPlugin, self).get_dataset_labels(dataset_obj)
-        user_id = None
-        if tk.current_user and tk.current_user.is_authenticated:
-            user_id = tk.c.userobj.id
-
-        capacity = authz.users_role_for_group_or_org(dataset_obj.owner_org, user_id)
-        is_org_admin = capacity == "admin"
-        # Editor shouldn't be able to collaborate on a dataset
-        if dataset_obj.creator_user_id != user_id and dataset_obj.state not in [
-            "active"
-        ] and not is_org_admin:
-            return []
+        # Override CKAN core
```

Comment:
By default, if a dataset is public CKAN would only return the public label for it. This way, besides the public label, it also adds the creator and collaborator labels to the dataset.